src/Nova/Room.php

```diff
@@ -17,7 +17,7 @@
 use Laravel\Nova\Panel;
 use Tipoff\Support\Nova\BaseResource;
 
-class Room extends BaseResource
+class Room extends BaseResource 
```

Comment:
hey @pdbreen could you help me here. in this nova I'm receiving 404 for cards and filters
![image](https://user-images.githubusercontent.com/13080491/115729070-17b0ae00-a353-11eb-80bb-09f72d9d28be.png)


 but I have information in the table

![image](https://user-images.githubusercontent.com/13080491/115728989-01a2ed80-a353-11eb-9b64-a6aeebc0c2ed.png)

the issue is only in this resource..